pkg/webhooks/webhooks.go

```diff
@@ -204,3 +206,16 @@ func HealthProbe(ctx context.Context) healthz.Checker {
 		return nil
 	}
 }
+
+func ValidateConversionEnabled(ctx context.Context, kubeclient client.Client) {
+	// allow context to exist longer than cache sync timeout
+	listCtx, cancel := context.WithTimeout(ctx, 130*time.Second)
+	defer cancel()
+	// sleep for cache hydration
+	// wait for cache to sync, controller-runtime defaults to 120 seconds
+	time.Sleep(120 * time.Second)
```

Comment:
Can we do something else here? a 120s sleep screams like we haven't thought about how to best wait for this to succeed.